src/modules/details/utils.ts

```diff
@@ -0,0 +1,15 @@
+import moment from 'moment';
+
+export const EtaButtonShow = ({
```

Comment:
please change the name of this util function. It should be generic name which can be used wherever we need this functionality